src/Microsoft.DotNet.ProjectModel/Compilation/LibraryExporter.cs

```diff
@@ -206,6 +213,74 @@ private IEnumerable<string> GetSharedSources(PackageDescription package)
                 .Where(path => path.StartsWith("shared" + Path.DirectorySeparatorChar))
                 .Select(path => Path.Combine(package.Path, path));
         }
+        
+        private IEnumerable<AnalyzerReference> GetAnalyzerReferences(PackageDescription package)
+        {
+            var analyzers = package
+                .Library
+                .Files
+                .Where(path => path.StartsWith("analyzers" + Path.DirectorySeparatorChar) &&
+                               path.EndsWith(".dll"));
+                               
+            var analyzerRefs = new List<AnalyzerReference>();
+            // See https://docs.nuget.org/create/analyzers-conventions for the analyzer
```

Comment:
We should use the nuget library for resolving this. Not manual path parsing (even tho thats what nuget does under the covers :smile: )